data/lib/src/datasource/shared_space_document_datasource.dart

```diff
@@ -95,10 +95,12 @@ abstract class SharedSpaceDocumentDataSource {
     Uri baseUrl);
 
   Future<bool> makeAvailableOfflineSharedSpaceDocument(
-    SharedSpaceNodeNested sharedSpaceNodeNested,
+    SharedSpaceNodeNested? sharedSpaceNodeNested,
```

Comment:
why not only `sharedSpaceId` only?

Reply:
We need to save this SharedSpace's information in the database too